src/app/technical-record/applicant/applicant.component.html

```diff
@@ -54,3 +54,13 @@
     </div>
   </dl>
 </ng-container>
+
+<ng-container *ngIf="editState">
```

Comment:
we need a test to cover the rendering of applicant editable component.

Reply:
done for other component as well

Reply:
done + other components
Thanks for spotting this out.